screen/settings/Shortcuts.tsx

```
@@ -0,0 +1,156 @@
+  const { setSelectedDefaultWallet, getSelectedDefaultWallet } = useOnAppLaunch();
+  const { receiveBitcoinIntent, setReceiveBitcoinIntent } = useSettings();
+  const [isSupportedEnvironment, setIsSupportedEnvironment] = useState(false);
+  const [defaultWallet, setDefaultWallet] = useState<TWallet | null>(null);
+
+  useEffect(() => {
+    const checkEnvironment = async () => {
+      const systemVersion = await DeviceInfo.getSystemVersion();
+      const majorVersionPart = systemVersion.split('.')[0];
+      const majorVersion = parseInt(majorVersionPart, 10);
+
+      if (isNaN(majorVersion)) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)